### DISCUSSION TITLE
Automatic Slicing

### DISCUSSION THREAD

**Discussion by TheBest6337** [OrcaSlicer/OrcaSlicer#970] [Ideas]
When a Model is in the build area, the model is sliced automatically, and you don't have to hit the slice button. This can be activated or deactivated by a button/switch.

**Comment by OtokoWa** [comment]
This feature would be amazing to have. When you tweak with the slicer setting it's amazing to have it sliced automaticaly. I use this feature with PrusaSlicer and SuperSlicer all the time and I kinda miss it in the OrcaSlicer.  +1

**Reply by foreachthing** [reply]
Yeah, but please for the love of "user-friendliness", create a toggle button/shortcut to enable/disable it quickly!!!

**Reply by MikMet** [reply]
Yes, this would be amazing.

**Reply by Throdne** [reply]
+1

**Reply by freakydude** [reply]
+1

**Reply by edooman** [reply]
+1

**Reply by caste1200** [reply]
+1

**Reply by jimwhitelaw** [reply]
+1

**Reply by Avpman2** [reply]
+1

**Reply by PedroC1999** [reply]
+1

**Reply by Mootronic** [reply]
+1

**Reply by Prog72** [reply]
+1

**Reply by AlekseyMko** [reply]
+1

**Reply by 1300371** [reply]
+1

**Comment by Deses** [comment]
Agreed. Cura has that and it's so convenient. This need to finds its way into OrcaSlicer.

**Reply by Avpman2** [reply]
+1

**Comment by mdeeter** [comment]
I deleted my +1 because using CTRL-R is just so easy.

**Comment by MichiMess** [comment]
+13

**Comment by C4Gamery** [comment]
I love this feature on other Slicers, as slicing is not super intensive and prevents 90% of the waiting for the slicing. Please add this feature!

**Comment by brian78675** [comment]
+1

**Comment by DafelixCly** [comment]
Maybe an option to make it time dependant? Something like "don't automatically slice again for 5 seconds" so it doesn't cause big performance issues when changing multiple settings in quick succession.

**Reply by freakydude** [reply]
Its already a feature of the 2.3.2 beta2

**Reply by DafelixCly** [reply]
Oh, thank you. I haven't seen it yet, I'll check better tomorrow
